2024-05-17 12:55:22,534 INFO    MainThread:547301 [wandb_setup.py:_flush():76] Current SDK version is 0.17.0
2024-05-17 12:55:22,535 INFO    MainThread:547301 [wandb_setup.py:_flush():76] Configure stats pid to 547301
2024-05-17 12:55:22,535 INFO    MainThread:547301 [wandb_setup.py:_flush():76] Loading settings from /home/tzikos/.config/wandb/settings
2024-05-17 12:55:22,535 INFO    MainThread:547301 [wandb_setup.py:_flush():76] Loading settings from /home/tzikos/Desktop/KULeuvenThesis/wandb/settings
2024-05-17 12:55:22,535 INFO    MainThread:547301 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-05-17 12:55:22,535 INFO    MainThread:547301 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-05-17 12:55:22,535 INFO    MainThread:547301 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program_relpath': 'main.py', 'program_abspath': '/home/tzikos/Desktop/KULeuvenThesis/main.py', 'program': '/home/tzikos/Desktop/KULeuvenThesis/main.py'}
2024-05-17 12:55:22,535 INFO    MainThread:547301 [wandb_setup.py:_flush():76] Applying login settings: {}
2024-05-17 12:55:22,535 INFO    MainThread:547301 [wandb_init.py:_log_setup():520] Logging user logs to /home/tzikos/Desktop/KULeuvenThesis/wandb/run-20240517_125522-zrxjkopt/logs/debug.log
2024-05-17 12:55:22,535 INFO    MainThread:547301 [wandb_init.py:_log_setup():521] Logging internal logs to /home/tzikos/Desktop/KULeuvenThesis/wandb/run-20240517_125522-zrxjkopt/logs/debug-internal.log
2024-05-17 12:55:22,535 INFO    MainThread:547301 [wandb_init.py:init():560] calling init triggers
2024-05-17 12:55:22,535 INFO    MainThread:547301 [wandb_init.py:init():567] wandb.init called with sweep_config: {}
config: {'learning_rate': 1e-06, 'batch_size': 64, 'L1': 0.0001, 'L2': 0.0001, 'A': 1, 'early_patience': 12, 'reduce_lr_patience': 5, 'architecture': 'ECGCNNClassifier', 'experiment': 'tachy', 'use_pretrained': False}
2024-05-17 12:55:22,536 INFO    MainThread:547301 [wandb_init.py:init():610] starting backend
2024-05-17 12:55:22,536 INFO    MainThread:547301 [wandb_init.py:init():614] setting up manager
2024-05-17 12:55:22,538 INFO    MainThread:547301 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2024-05-17 12:55:22,540 INFO    MainThread:547301 [wandb_init.py:init():622] backend started and connected
2024-05-17 12:55:22,545 INFO    MainThread:547301 [wandb_init.py:init():711] updated telemetry
2024-05-17 12:55:22,551 INFO    MainThread:547301 [wandb_init.py:init():744] communicating run to backend with 90.0 second timeout
2024-05-17 12:55:23,040 INFO    MainThread:547301 [wandb_run.py:_on_init():2396] communicating current version
2024-05-17 12:55:23,126 INFO    MainThread:547301 [wandb_run.py:_on_init():2405] got version response 
2024-05-17 12:55:23,127 INFO    MainThread:547301 [wandb_init.py:init():795] starting run threads in backend
2024-05-17 12:55:25,830 WARNING MainThread:547301 [wandb_init.py:init():1168] interrupted
Traceback (most recent call last):
  File "/home/tzikos/anaconda3/envs/KUThesis/lib/python3.12/site-packages/wandb/sdk/wandb_init.py", line 1164, in init
    return wi.init()
           ^^^^^^^^^
  File "/home/tzikos/anaconda3/envs/KUThesis/lib/python3.12/site-packages/wandb/sdk/wandb_init.py", line 808, in init
    run_start_result = run_start_handle.wait(timeout=30)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/tzikos/anaconda3/envs/KUThesis/lib/python3.12/site-packages/wandb/sdk/lib/mailbox.py", line 283, in wait
    found, abandoned = self._slot._get_and_clear(timeout=wait_timeout)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/tzikos/anaconda3/envs/KUThesis/lib/python3.12/site-packages/wandb/sdk/lib/mailbox.py", line 130, in _get_and_clear
    if self._wait(timeout=timeout):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/tzikos/anaconda3/envs/KUThesis/lib/python3.12/site-packages/wandb/sdk/lib/mailbox.py", line 126, in _wait
    return self._event.wait(timeout=timeout)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/tzikos/anaconda3/envs/KUThesis/lib/python3.12/threading.py", line 655, in wait
    signaled = self._cond.wait(timeout)
               ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/tzikos/anaconda3/envs/KUThesis/lib/python3.12/threading.py", line 359, in wait
    gotit = waiter.acquire(True, timeout)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
KeyboardInterrupt
2024-05-17 12:55:29,528 WARNING MsgRouterThr:547301 [router.py:message_loop():77] message_loop has been closed
